index.js

```diff
@@ -50,6 +51,11 @@ module.exports = options => {
 			subprocess.stderr.pipe(process.stderr);
 		}
 
+		if (options.outputfile) {
```

Comment:
```suggestion
		if (options.outputFile) {
```

And the option needs to be documented in the readme.